packages/connect/e2e/tests/device/cancelCoinjoinAuthorization.test.ts

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable import/no-named-as-default */
+
+import TrezorConnect, { Success, PROTO, Unsuccessful } from '../../../src';
+
+const { ADDRESS_N } = global.TestUtils;
+const { getController, setup, conditionalTest, initTrezorConnect } = global.Trezor;
+
+const controller = getController('applyFlags');
+
+describe('TrezorConnect.cancelCoinjoinAuthorization', () => {
+    beforeAll(async () => {
+        await setup(controller, {
+            mnemonic: 'mnemonic_all',
+        });
+    });
+
+    beforeEach(async () => {
+        // restart connect for each test (working with event listeners)
+        await TrezorConnect.dispose();
+
+        await initTrezorConnect(controller, { debug: false });
+
+        TrezorConnect.on('DEVICE_EVENT', ev => {
+            if ('code' in ev.payload && ev.payload.code === 'ButtonRequest_PinEntry') {
+                controller.send({ type: 'emulator-input', value: '1234' });
+            }
+        });
+    });
+
+    afterAll(async () => {
+        controller.dispose();
+        await TrezorConnect.dispose();
+    });
+
+    conditionalTest(['1', '<2.5.4'], 'Cancel authorization works', async () => {
+        // const result = await TrezorConnect.cancelCoinjoinAuthorization({});
+
+        // expect(result.success).toBe(true);
+        // expect((result as Success<PROTO.Success>).payload.message).toBe('Authorization cancelled');
+
+        //
+        //
+        //
+        //
+
+        // unlocked path is required for tx validation
```

Comment:
not sure if it is required? is it? maybe it is only required for signTransaction?
pls try it out without it

+ remove commented block above

Reply:
5780f7c05